src/sass_context_wrapper.cpp

```diff
@@ -1,3 +1,4 @@
+#include <stdlib.h>
```

Comment:
I know this is a standard header but this is only required for freebsd at the moment. On other platforms, gcc is compiling without it.